api/src/components/coauthor/schema/updateAll.ts

```diff
@@ -26,6 +26,12 @@ const createCoAuthorSchema: I.Schema = {
             linkedUser: {
                 type: ['string', 'null']
             },
+            createdAt: {
```

Comment:
Yes we pass them to the api. We have that "additionalProperties: false" flag to not allow any extra properties. If we don't put them here, then they are treated as additional properties and the backend will return a validation error... This endpoint handles a PUT request, that means we pass the updated data (from the client) in the same format we received it (from the backend).

Reply:
will be very confusing to anyone not aware of that, passing fields that are simply ignored cannot be good practice, if this were a public API I would definitely require them not to be there, the UI should send the correct data (which we do for the publication) and suppress sending those as they are read-only.

Reply:
But we are sending the correct data. We cannot take user input for 'createdAt' and 'reminderDate' since 'createdAt' is auto-generated by the DB at the creation time and reminderDate is updated by the backend only, after sending a reminder. I don't see anything confusing here. It is normal to not allow co-authors or main author to update other co-authors info as they wish